scripts/server

```diff
@@ -12,17 +12,26 @@ Starts servers using docker-compose
 "
 }
 
-
 if [ "${BASH_SOURCE[0]}" = "${0}" ]
 then
     if [ "${1:-}" = "--help" ]
     then
         usage
+    elif [ "$1" = "--no-airflow" ]
```

Comment:
I think so? It has sounded like `./scripts/server` is "just gimme everything" while the `--no-airflow` option was desirable so we would have a "just gimme everything but not _everything_ everything".